core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2452,7 +2452,7 @@ package object config {
       .createOptional
 
   private[spark] val JAR_IVY_SETTING_PATH =
-    ConfigBuilder("spark.jars.ivySettings")
+    ConfigBuilder(MavenUtils.JAR_IVY_SETTING_PATH_KEY)
```

Comment:
Isn't it more appropriate to have `MavenUtils` refer to this config entry and get the key name?

It seems weird and breaks with the remaining configs for this config to get its key name from somewhere else.